boards/croxel/croxel_cx1825/croxel_cx1825_nrf52840.yaml

```
@@ -14,4 +14,5 @@ supported:
   - gpio
   - i2c
   - watchdog
+  - hts221
```

Likewise, this should be:
```suggestion
  - ambient_temp
  - humidity
```

---

ditto